apps/mobile/mobile-app/components/Filter.tsx

```
@@ -0,0 +1,40 @@
+import React from 'react';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)